content/docs/Core principles/what-is-beehaw.md

```diff
@@ -3,25 +3,25 @@ title: What is Beehaw?
 
 Many philosophers have written extensively about this subject, and we simply don’t have time to write an entire manifesto. In simple terms, we are not advocating for tone policing. We believe that being outraged and angry at people who are destroying our society is a good thing to do. When the Supreme Court removes protections for abortion, it’s okay to be outraged and to take action into your own hands - they have done something intolerant. When someone advocates online that you don’t have the right to your own body, it’s okay to tell them to fuck off. In fact, we greatly encourage it. This is **being intolerant to the intolerant**.
 
 *However*, when someone online shares an opinion and it feels like they might be intolerant and you jump to the conclusion that they are intolerant and you launch into a tirade at them, this is not nice behavior. You didn’t check if they have the opinion you think they have, and that’s simply not nice to someone which you don’t know.
 
-It gets even more complicated when you consider someone who is sharing an opinion they have which is actively harmful to many individuals in the world, but it’s due to their *ignorance*. We personally believe that so long as this person is not actively spreading this intolerant viewpoint and are working on themselves to become a better person, that it would not be particularly productive to launch into a tirade against them. We understand, however, how someone could be quite rude in response to such intolerance and we agree that this person may desperately need to be educated appropriately, but there is no way for that discussion to happen on this platform in a productive manner while lobbing insults at each other. We can understand why, at first brush, some might consider this tone policing. This is all to say, we disapprove of the intolerant viewpoint, and we approve of it being corrected, but we also approve of the intolerant person attempting to become a better person.
+It gets even more complicated when you consider someone who is sharing an opinion they have which is actively harmful to many individuals in the world, but it’s due to their *ignorance*. We personally believe that so long as this person is not actively spreading this intolerant viewpoint and is working on themself to become a better person, that it would not be particularly productive to launch into a tirade against them. We understand, however, how someone could be quite rude in response to such intolerance and we agree that this person may desperately need to be educated appropriately. However, there is no way for that discussion to happen on this platform in a productive manner while lobbing insults at each other. We can understand why, at first brush, some might consider this tone policing. This is all to say: we disapprove of the intolerant viewpoint, and we approve of it being corrected, but we also approve of the intolerant person attempting to become a better person.
```

Comment:
Is themself a neo-pronoun? I'm not really familiar with this one and it might be easier to just pluralize the entire sentence.

Reply:
Well, it is legit I think (see for example https://www.merriam-webster.com/words-at-play/Themself) but there's an easy fix to avoid its use altogether so I just dropped it. I don't want to switch the sentence to plural because then it would conflict with "this person" in the previous sentence.